src/components/FilmTime/FilmTimeHour.js

```diff
@@ -0,0 +1,21 @@
+const FilmTimeHour = [
```

Comment:
zastanów się nad nazwami zmiennych. Czy następne obiekty to naprawdę godziny filmów czy może coś więcej?